mods/ts/rules/civilian-structures.yaml

```diff
@@ -1213,6 +1200,33 @@ GAOLDCC6:
 	RenderBuilding:
 		Palette: player
 
+GASAND:
+	Inherits: ^Wall
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 1000
+		Prerequisites: ~disabled
```

Comment:
This had `~structures.gdi` before. Move it to `gdi-defenses.yaml` and fix the prerequisite?